src/Core/Framework/Adapter/Command/CacheClearAllCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Command;
+
+use Shopware\Core\Framework\Adapter\Cache\CacheClearer;
+use Shopware\Core\Framework\Adapter\Console\ShopwareStyle;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[Package('core')]
+#[AsCommand(name: 'cache:clear:all', description: 'Clear all caches/pools')]
```

Comment:
Can you maybe elaborate a little in the description about why this is different to the other options?

Reply:
Hi @AydinHassan, I added a command help and more complete description

Reply:
thanks!